### DISCUSSION TITLE
Documentation missing

### DISCUSSION THREAD

**Discussion by gallwood** [BlazorFluentUI/BlazorFluentUI#346] [Q&A] [answered]
Hi, I'n new here. Just wanted to try the Blazor components out. 

I like what I have seen so far.

I have seen the Getting Started WIKI page which is great. It also gives a link to the Fluent UI web page (Microsoft). Is this the documentation I should be using?

Just one more thing, whilst on the subject of documentation. I was looking at the source code for the demo site for this repo. Some razor components use a ResponsiveLayout. I see this is in the BlazorFluentUI library, but is there documentation for this?

Thanks

Graham

**Comment by limefrogyank** [comment, marked as answer]
We don't have documentation specifically for the components other than any comments on the demo pages for each component.  We try to achieve parity with the react components which is why (for now) we tell people to look at their documentation to get an idea of what you can do/should do.  But it's far from perfect.

Ideally, we'd have autogenerating docs pulled from comments on the code itself.  I know stuff like that exists, but I've never used any myself.  We're always looking for help so if you know a good method to do this, we'd love the help...  you wouldn't need to write the comments, just setup a framework so that we can start doing it in a consistent manner. 😁

**Reply by limefrogyank** [reply]
FWIW, ResponsiveLayout was something I made a long time ago to deal with changing screen widths without having to use CSS.  Just wrap your components in a ResponsiveLayout and set the parameters and it will auto generate the css media queries that show or hide that section.

**Reply by gallwood** [reply]
Thanks for the reply. That all makes sense. As for the code comments -> Documentation, I understand your thinking too. I'll have a think about the problem.

So, is the FluentUI mainly for apps that are going to extend the Microsoft 365 apps or can anyone use it for any type of app?

**Reply by limefrogyank** [reply]
It's for anything.  There are some restrictions when using some of the icons... there's a link somewhere, but they're pretty lenient.  I'm betting they're just picky about how the brand icons (Excel, Word, etc) are used.  

This project is NOT a microsoft project.  We're just copying their style for Blazor.
